modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/JavaServiceTaskDeploymentUndeploymentTestCase.java

```diff
@@ -0,0 +1,212 @@
+            log.info(jsonException.getMessage());
+            Assert.fail();
+        }
+
+
+        //Suspending the Process Instance
+        try {
+            String[] suspendResponse = tester.suspendProcessInstanceById(processInstanceResponse[1]);
+            Assert.assertTrue("Process Instance has been suspended", suspendResponse[0].contains(BPMNTestConstants.OK));
+            Assert.assertTrue("Process Instance has been suspended", suspendResponse[1].contains("true"));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        } catch (JSONException jsonException) {
+            log.info(jsonException.getMessage());
+            Assert.fail();
+        }
+
+
+        try {
+            String stateVerfication = tester.getSuspendedStateOfProcessInstanceByID(processInstanceResponse[1]);
+            Assert.assertTrue("Verifying Suspended State", stateVerfication.contains("true"));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        } catch (JSONException jsonException) {
+            log.info(jsonException.getMessage());
+            Assert.fail();
+        }
+
+
+        //Deleting a Process Instance
+        try {
+            String deleteStatus = tester.deleteProcessInstanceByID(processInstanceResponse[1]);
+            Assert.assertTrue("Process Instance Removed", deleteStatus.contains(BPMNTestConstants.NO_CONTENT));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        }
+
+        try {
+            String deleteCheck = tester.validateProcessInstanceById(definitionResponse[1]);
+            Assert.assertTrue("Process Instance Removed Check", deleteCheck.contains(BPMNTestConstants.NOT_AVAILABLE));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        } catch (JSONException jsonException) {
+            log.info(jsonException.getMessage());
+            Assert.fail();
+        }
+
+
+        //Deleting the Deployment
+        try {
+            String undeployStatus = tester.unDeployBPMNPackage(deploymentResponse[1]);
+            Assert.assertTrue("Package UnDeployed", undeployStatus.contains(BPMNTestConstants.NO_CONTENT));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        }
+
+        try {
+            String[] unDeployCheck = tester.getDeploymentInfoById(deploymentResponse[1]);
+            Assert.assertTrue("Package UnDeployment", unDeployCheck[0].equals(BPMNTestConstants.NOT_AVAILABLE));
+        } catch (IOException ioException) {
+            log.info(ioException.getMessage());
+            Assert.fail();
+        } catch (JSONException jsonException) {
+            log.info(jsonException.getMessage());
+            Assert.fail();
```

Comment:
If both exception cases are handled in the same way, use a single "Exception" catch block.